### DISCUSSION TITLE
[Feature Request]: Consider using Continuous Localization

### DISCUSSION THREAD

**Discussion by s3nnet** [GetPublii/Publii#1778] [Ideas]
### Feature Description

Transifex - Weblate - Codeberg - u name it.
In turn I would consider taking care of your German translation.
Find my Transifex profile over at https://app.transifex.com/user/profile/Gyges/

**Comment by dziudek** [comment]
Hi,

At this moment we have no plans and time slots to add, migrate and manage additional platform to handle localizations. Maybe in the future - I am converting this issue to a discussion

**Comment by s3nnet** [comment]
Would a contact to a nice Weblate technician help? In this case I could help!
